### PR TITLE
add fingerprint expedition project

### PR DESCRIPTION
**Pull request by nmasdoufi-ol**
image

### CODE REVIEW

File: plugins/fortinet_fortipam.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "FortiPAM provides tightly controlled privileged access to the most sensitive resources within an organization. "
+  website "https://www.fortinet.com/products/fortipam"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /FortiPAM/,
```
Comment (ybadaoui-ostorlab) [inline plugins/fortinet_fortipam.rb:13]: this won't give you some FPs?

File: plugins/fortinet_fortiproxy.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "FortiProxy delivers broad protection and visibility to every network segment, device, and appliance, whether deployed virtually, in the cloud, or on-premises."
+  website "https://www.fortinet.com/products/secure-web-gateway/fortiproxy"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /FortiProxy/,
```
Comment (ybadaoui-ostorlab) [inline plugins/fortinet_fortiproxy.rb:13]: as well as this i think!

File: plugins/fortinet_fortiswitchmanager.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "FortiSwitch Manager enables network administrators to cut through the complexities of non-FortiGate-managed FortiSwitch deployments."
+  website "https://docs.fortinet.com/product/fortiswitch-manager"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /FortiSwitchManager/,
```
Comment (ybadaoui-ostorlab) [inline plugins/fortinet_fortiswitchmanager.rb:13]: this one also, if you can find something more special, it's very easy to find those words in the body of other websites.

**Review by elyousfi5 — APPROVED** [review]

File: plugins/fortinet_fortipam.rb (hunk 1 above)
Comment (nmasdoufi-ol) [reply plugins/fortinet_fortipam.rb]: For this one I was able to find a more accurate fingerprint, but for the others nothing.

File: plugins/expedition_project.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "Expedition Project"
```
Comment (3asm) [inline plugins/expedition_project.rb:2]:
```suggestion
  name "Palo Alto Expedition Project"
```

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by deadly-panda — APPROVED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**